test/TestServerProjectsLowLevel/media_types/Generated/MediaTypesClient.cs

```diff
@@ -51,7 +51,7 @@ public MediaTypesClient(AzureKeyCredential credential, Uri endpoint = null, Medi
 
         /// <summary> Analyze body, that could be different media types. </summary>
         /// <param name="content"> The content to send as the body of the request. </param>
-        /// <param name="contentType"> Upload file type. Allowed values: &quot;application/pdf&quot; | &quot;image/jpeg&quot; | &quot;image/png&quot; | &quot;image/tiff&quot;. </param>
+        /// <param name="contentType"> Upload file type. Allowed values: &quot;application/json&quot; | &quot;application/pdf&quot; | &quot;image/jpeg&quot; | &quot;image/png&quot; | &quot;image/tiff&quot;. </param>
```

Comment:
Why is Json added here?  Did the swagger file change?

Reply:
No. It is because:
Previously, we have two methods: json and others.
Now after the M4 change, we only have one method: containing json and others.
So the change here is actually: remove the json one and add the json type to the remaining one.